fidesctl/src/fidesapi/database/crud.py

```diff
@@ -146,9 +146,8 @@ async def upsert_resources(
                     log.debug("Upserting resources")
                     insert_stmt = _insert(sql_model).values(resource_dicts)
                     await session.execute(
-                        insert_stmt.on_conflict_do_update(
+                        insert_stmt.on_conflict_do_nothing(
```

Comment:
I'm very much on the fence with this change here, I think it sort of defeats the purpose of using upsert except in the case of the default taxonomy. As far as I can tell however, it is the only use we have of upsert as of today... Would love an outside opinion here

Reply:
if someone runs `fidesctl db reset` we still want to give them a clean slate, but otherwise i don't think the upsert should be overwriting anything the user has done, and therefore I think this change makes sense.

so to be clear on what I think is okay here:
* `fidesctl db reset` wipes the whole database and adds all new stuff as per usual
* spinning up the webserver should _not_ overwrite what's already in the database (by using the on_conflict_do_nothing)

does this make sense/sound right?

Reply:
Thanks for the second opinion! Yes that makes sense to me and I confirmed it works as you stated 🙌🏽

Reply:
perfect, thank you!